deployment/docker-compose/pretix/pretix/etc/pretix.cfg

```diff
@@ -0,0 +1,28 @@
+[pretix]
+instance_name=My pretix installation
+url=http://127.0.0.1/
```

Comment:
There should be a note in the Readme to change this line to allow access from external. Otherwise the user receives a django Host Denial `django.security.DisallowedHost exception Unknown host:`